packages/e2e-tests/specs/__snapshots__/block-transforms.test.js.snap

```diff
@@ -0,0 +1,183 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`test transforms individual transforms work as expected Code block should transform to Preformatted block. fixture: core__code 1`] = `
```

Comment:
This doesn't read very well. Ideally, it should be a readable sentence. E.g. `RichText should insert character`.

Reply:
Is there any way to remove or reduce `test transforms individual transforms work as expected `?